csharp/codeql-extractor.yml

```diff
@@ -16,3 +16,12 @@ file_types:
     extensions:
       - .cs
 legacy_qltest_extraction: true
+options:
+  buildless:
+    title: Whether to use buildless (standalone) extraction.
+    description: >
+      A value indicating, which type of extraction the autobuilder should perform.
+      If 'yes', then the standalone extractor will be used, otherwise tracing extraction
```

Comment:
I think we should use `true` instead of `yes` (and `false` below instead of `no`).